### ISSUE TITLE
Move build infra from Centos7 to Rocky9

### ISSUE THREAD

**Issue by lungdear** [OpenSDN-io/community#7]
Move build infra from centos7 to rocky9

**Comment by randybias** [comment]
Please update with latest status.

**Comment by Andrey-mp** [comment]
first step - base OS has been switched from centos7 to rocky9

next step is to use rocky9 as a base fo rtf-dev-env. this step will be rather big. I will post updates here.

**Comment by randybias** [comment]
Updates @Andrey-mp

**Comment by Andrey-mp** [comment]
this task is related and depends on several tasks: c++11 and python3 (in general this task is blocked by other)

**Comment by randybias** [comment]
This is done now, correct?

**Comment by Andrey-mp** [comment]
No. It's a most long task - it depends on python3 and c++11. Only then we can switch build to rocky9 from centos7

**Comment by mkraposhin** [comment]
> No. It's a most long task - it depends on python3 and c++11. Only then we can switch build to rocky9 from centos7

Right now, approximately 95% of the code uses C++11 except CQL driver. I would postpone transition of CQL driver to C++11 till the next release.

**Comment by randybias** [comment]
Why postpone the transition of the CQL driver to C++11?

**Comment by mkraposhin** [comment]
> Why postpone the transition of the CQL driver to C++11?

Because this is something that can't be done in a hurry. I have to understand in details how it works. The current implementation of the CQL driver extensively uses auto_ptr + boost function pointers which is not possible with unique_ptr which must be used instead of auto_ptr.

**Comment by randybias** [comment]
OK.  Can I get some kind of ETA?  Moving to rel2.  I will ask Alexander Ignatov if some of his team could help.

**Comment by Andrey-mp** [comment]
Yeah, it should be moved to rel2. We didn't start and can't suppose what hidden problems can arise.
